src/modules/did/did.service.ts

```diff
@@ -322,18 +322,20 @@ export class DIDService implements OnModuleInit, OnModuleDestroy {
   }
 
   /**
-   * Resolves document service endponts
+   * Resolves service endponts, which represent credentials
    *
    * @param did DID of the document service endpoints
    */
-  public async resolveServiceEndpoints(did: string) {
+  public async resolveCredentials(did: string) {
     const { service } = await this.getById(did);
-    return Promise.all(
-      service
-        .map(({ serviceEndpoint }) => serviceEndpoint)
-        .filter((endpoint) => IPFSService.isCID(endpoint))
-        .map((cid) => this.ipfsService.get(cid))
-    );
+    return (
+      await Promise.all(
+        service
+          .map(({ serviceEndpoint }) => serviceEndpoint)
+          .filter((endpoint) => IPFSService.isCID(endpoint))
+          .map((cid) => this.ipfsService.get(cid).catch(() => null))
```

Comment:
@JGiter what do you think of adding a log of the error here?
```suggestion
          .map((cid) => this.ipfsService.get(cid).catch((err) => {
            this.logger.error(`Error resolving ${cid} for ${did}: ${err}`)
            return null
          }))
```

Reply:
It is helpful indeed. I suggest to add it to IpfsService.get though so that we don't log in every place, where getting from IPFS. I added this in next commit